Raven.Tests.Helpers/RavenTestBase.cs

```diff
@@ -138,6 +138,24 @@ protected string NewDataPath(string prefix = null, bool forceCreateDir = false)
 			return newDataDir;
 		}
 
+        /// <summary>
+        /// Creates a new Embeddable document store.
+        /// </summary>
+        /// <param name="runInMemory">The document store is all stored in RAM. Nothing hits the file system. (Default is True).</param>
+        /// <param name="requestedStorage"></param>
+        /// <param name="catalog"></param>
```

Comment:
These are custom bundles? Link to an example how this is setup (how i would create an instance, which is to be passed in).